x-pack/solutions/security/plugins/security_solution/server/lib/entity_analytics/entity_store/elasticsearch_assets/updates_entity_index.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ElasticsearchClient, Logger } from '@kbn/core/server';
+import {
+  ENTITY_UPDATES,
+  ENTITY_SCHEMA_VERSION_V1,
+  entitiesIndexPattern,
+} from '@kbn/entities-schema/src/schema/v1/patterns';
+import { type EntityType } from '../../../../../common/api/entity_analytics/entity_store';
+import { createEntityIndex, deleteEntityIndex } from './entity_index';
+
+export const createEntityUpdatesIndex = async (
+  entityType: EntityType,
+  esClient: ElasticsearchClient,
+  namespace: string,
+  logger: Logger
+) => {
+  await createEntityIndex({
```

Comment:
Is there a benefit of calling the modified `createEntityIndex()` here instead of `esClient.indices.create()`? I'm asking primarily because I'm a bit worried of adding the `indexName` override to the function parameters. I'm sure it works fine, just looks like a potential foot gun.